backend/migrate_imports_dates.py

```
@@ -0,0 +1,161 @@
+    if account.imports_end_date is None and account.last_transaction_date is not None:
+        update_data['imports_end_date'] = account.last_transaction_date
+        logger.debug(f"Setting importsEndDate to {account.last_transaction_date}")
+    
+    # Update the account if we have changes
+    if update_data:
+        try:
+            updated_account = update_account(account.account_id, account.user_id, update_data)
+            logger.info(f"Updated account {account.account_id} ({account.account_name}) - "
+                       f"importsStartDate: {updated_account.imports_start_date}, "
+                       f"importsEndDate: {updated_account.imports_end_date}")
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)